homeassistant/components/modbus.py

```diff
@@ -26,6 +26,7 @@
 CONF_STOPBITS = 'stopbits'
 CONF_TYPE = 'type'
```

Comment:
That one is already defined in `const.py`. I know that it's not your code but would be nice if you could fix it as well.